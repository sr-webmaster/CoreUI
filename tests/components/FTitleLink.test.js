import { mount } from '@vue/test-utils'
import Component from '@freshinup/core-ui/src/components/FTitleLink.vue'

describe('FTitleLink', () => {
  it('snapshot default', async () => {
    const wrapper = mount(Component)
    expect(wrapper.element).toMatchSnapshot()
  })
  it('snapshot with color', async () => {
    const wrapper = mount(Component, {
      propsData: {
        color: 'info'
      }
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
