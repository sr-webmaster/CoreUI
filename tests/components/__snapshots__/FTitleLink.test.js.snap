// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FTitleLink snapshot default 1`] = `
<a
  class="title primary--text text-truncate"
/>
`;

exports[`FTitleLink snapshot with color 1`] = `
<a
  class="title info--text text-truncate"
/>
`;
